discojs/discojs-core/src/models/gpt/model.ts

```diff
@@ -0,0 +1,542 @@
+import tf, { LayersModel, layers, serialization } from '@tensorflow/tfjs'
```

Comment:
comment that this whole file (or most?) is from `gpt-tfjs`? (or is it?)

Reply:
Indeed it is, everything under gpt/ is a modification or straight up copy paste from gpt-tfjs.